repo2docker/buildpacks/r.py

```diff
@@ -19,7 +19,14 @@ class RBuildPack(PythonBuildPack):
        date snapshot of https://mran.microsoft.com/timemachine
        from which libraries are to be installed.
 
-    2. An optional `install.R` file that will be executed at build time,
+    2. A `DESCRIPTION` file signaling an R package, or
```

Comment:
Maybe we should split this discussion into two things: how we detect that R needs installing and how to specify which libraries should be installed.

A more eloquent version of:
```
Setup R + RStudio if runtime.txt or DESCRIPTION or jats thing

Libraries will be installed if specified in install.R or as dependencies in DESCRIPTION
```